Utilities/DataLoading/EdFi.SmokeTest.Console/Program.cs

```diff
@@ -79,13 +87,30 @@ private static void StartCommandLine(Container container)
 
         private static void ConfigureCommandLineContainer(Container container, Configuration configuration)
         {
+            var hostBuilder = new HostBuilder()
```

Comment:
confused why we are building a new host here.

Reply:
rolled back code